### DISCUSSION TITLE
GitHub Universe 🪐 Cloud Skills Challenge

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#69709] [Copilot News and Announcements]
![Screenshot 2023-10-10 at 8 56 07 PM](https://github.com/community/community/assets/95834130/f6b7953a-4f72-4345-ae6b-b52f6fcc6bc7)

Microsoft and GitHub have just launched 🚀  an initiative for everyone who is ready to start their AI Journey – **GitHub Universe Cloud Skills Challenge**. Throughout November there will be a self-study challenge designed to allow you to explore some of the most requested GitHub tools for AI beginners, and to explore new opportunities.

The GitHub Universe Cloud Skills Challenge is a 30-day learning challenge on Microsoft Learn. It is a free, fun, community-oriented program that will help you learn GitHub, Codespaces 💻 , and GitHub Copilot 🤖  . You will learn how to setup your development environment in the cloud with Codespaces and create real-world projects with GitHub Copilot, **[right in time for GitHub Universe! 🪐 ](https://githubuniverse.com/)**

With easy-to-follow lessons and hands-on exercises, you'll get to know the amazing tools that you might need to pursue a career in this fast-growing field. 

## [Register now for the challenge and start learning more about AI](https://aka.ms/GitHubUniverseCSC)

By completing this challenge, you will be able to obtain a digital badge for having completed this experience.

For more information about the challenge, visit our [announcement](https://techcommunity.microsoft.com/t5/educator-developer-blog/announcing-github-universe-cloud-skills-challenge/ba-p/3943877?WT.mc_id=academic-109090-cyzanon). ⚓

**Comment by Nahid4306053** [comment]
I'm glad to hear you're interested in the GitHub Universe Cloud Skills Challenge! It's a fantastic opportunity to dive into the world of AI and learn about GitHub tools. To get started, follow these steps:

Visit the GitHub Universe Cloud Skills Challenge Page: Go to the GitHub Universe Cloud Skills Challenge page on Microsoft Learn to find all the necessary information and resources.

Register for the Challenge: Sign up for the challenge to officially participate. This will likely involve creating an account or logging in if you already have one.

Explore GitHub Tools: Throughout November, engage with the self-study materials provided by the challenge. Familiarize yourself with GitHub, Codespaces, and GitHub Copilot. These tools are essential for any AI enthusiast or developer.

Complete Hands-on Exercises: Make sure to complete the hands-on exercises to gain practical experience. Learning by doing is one of the most effective ways to understand these tools.

Connect with the Community: Engage with the community of learners. You can ask questions, share your progress, and learn from others' experiences. Community support can be invaluable during a learning journey.

Earn Your Digital Badge: Upon completing the challenge successfully, you'll earn a digital badge. This badge serves as a certification of your accomplishment and can be a valuable addition to your professional profile.

Stay Updated: Keep an eye on announcements and updates related to the challenge. Stay engaged with the community even after the challenge ends. Continuous learning is key in the rapidly evolving field of AI and technology.

Remember, the journey of learning is as important as the destination. Enjoy the process, and best of luck with your AI learning adventure! 🚀

**Comment by safnimj** [comment]
Just conquered the challenge and crushed it! 🔥✨ I have officially completed it, and it feels amazing! 💪😎 Any extra compliments or swags coming my way? 😉

**Comment by damicodedotpy** [comment]
How do I do to claim my badge? :( I already finished

**Reply by wibrvallejoma** [reply]
same here! :(
